lib/testmap.py

```diff
@@ -49,7 +49,8 @@ def contexts(image, *scenarios: List[str]):
             # no udisks on CoreOS → skip storage
             *contexts('fedora-coreos', COCKPIT_SCENARIOS - {'storage'}),
             *contexts('rhel-8-9', COCKPIT_SCENARIOS),
-            *contexts('rhel-8-9-distropkg', COCKPIT_SCENARIOS),
+            # all skipped
+            *contexts('rhel-8-9-distropkg', COCKPIT_SCENARIOS - {'networking'}),
```

Comment:
I'm not a huge fan of how brittle this is.  We skip this because we know that there are no tests there, but maybe one day we do have tests there and falsely conclude that they're passing...

I understand that this is a temporary measure, but ya... let's try to figure out something better asap.